server/storage/interface.go

```diff
@@ -39,4 +39,15 @@ type MetaStore interface {
 	// Delete removes all metadata for a given GUN.  It does not return an
 	// error if no metadata exists for the given GUN.
 	Delete(gun string) error
+
+	// GetChanges returns an ordered slice of changes. It starts from
+	// the change matching changeID, but excludes this change from the results
+	// on the assumption that if a user provides an ID, they've seen that change.
+	// If changeID is 0, it starts from the
```

Comment:
@endophage Ah right, forgot about that part of the discussion.  I'm not sure how to indicate that then with a string, if we want to keep them opaque IDs... if they always had to evaluate to an integer, to indicate "latest", "earliest", or otherwise a totally valid change ID, would it make sense for them to just be integers instead of strings, if they always had to evaluate that way?